src/reference/informative/informC04_component_reference.rst

```diff
@@ -8,4 +8,197 @@
 
   .. container:: infospec
 
-    **TODO**
+    .. container:: heading3
+
+      Understanding component references
+
+    As with references to units and variables, the term "component reference" refers to places where you need to  refer to a component using its name.
+    This is done through the :code:`component_ref` attribute found in :code:`import component` and :code:`encapsulation` elements, as well as the :code:`component_1` and :code:`component_2` attributes of :code:`connection` elements.
```

Comment:
We don't have something similar for the informative units reference documentation. So, should this be dropped or should something similar be added to the informative units reference documentation? I think I would like the same to be done for units references (after the reordering).

Reply:
I think we do, but it will be in another PR, so in another branch of the docs?  See: https://cellml-specification-dev.readthedocs.io/en/i137_inform3_2/reference/formal_and_informative/specC02_units_reference.html